client/app/src/captain-vote/captain-vote.html

```diff
@@ -2,4 +2,9 @@
 	<h2>Election du capitaine</h2>
 
 	<player-vote></player-vote>
+
+	<div *ngIf="isMJ">
+		<button (click)="next()" class="btn btn-primary">Continuer</button>
+	</div>
+
 </div>
```

Comment:
Configure ton Webstorm pour ajouter les EOF au save :

Settings --> General --> Other --> "Ensure line feed at file end on Save"

http://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file